README.md

```diff
@@ -23,10 +71,59 @@ Additionally, Kuadrant provides the following CRDs
.629629629629633,0)" xlink:href="#aP"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,31.506172839506174,0)" xlink:href="#aM"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,41.38271604938272,0)" xlink:href="#bh"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,52.19753086419754,0)" xlink:href="#aO"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,62.074074074074076,0)" xlink:href="#bi"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,72.8888888888889,0)" xlink:href="#bj"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,77.82716049382717,0)" xlink:href="#aQ"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,83.70370370370371,0)" xlink:href="#aP"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,93.58024691358025,0)" xlink:href="#aN"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,100.49382716049384,0)" xlink:href="#aW"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,111.30864197530865,0)" xlink:href="#aO"/><use transform="matrix(0.04938271604938272,0,0,0.04938271604938272,121.18518518518519,0)" xlink:href="#aQ"/></g><path fill="#333" d="M115-194c53 0 69 39 70 98 0 66-23 100-70 100C84 3 66-7 56-30L54 0H23l1-261h32v101c10-23 28-34 59-34zm-8 174c40 0 45-34 45-75 0-40-5-75-45-74-42 0-51 32-51 76 0 43 10 73 51 73" id="bk"/><path fill="#333" d="M0 4l72-265h28L28 4H0" id="bl"/><g id="ae"><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,0,0)" xlink:href="#bc"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,10.370370370370368,0)" xlink:href="#aq"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,19.01234567901234,0)" xlink:href="#bk"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,27.654320987654316,0)" xlink:href="#as"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,36.29629629629629,0)" xlink:href="#bl"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,40.61728395061728,0)" xlink:href="#af"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,52.71604938271604,0)" xlink:href="#ah"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,61.35802469135801,0)" xlink:href="#as"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,69.99999999999999,0)" xlink:href="#av"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,78.64197530864196,0)" xlink:href="#ag"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,89.01234567901233,0)" xlink:href="#ai"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,97.6543209876543,0)" xlink:href="#al"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,101.06790123456788,0)" xlink:href="#aJ"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,105.38888888888887,0)" xlink:href="#ar"/></g></defs></g></svg>
```

Comment:
is there a way to remove the raw content from the README?? put that in a svg file and make a reference  from the README?